Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -687,10 +696,12 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
                     try await self.handleAdapterStarted(startReason: startReason)
 
                     // Enable Connect on Demand when manually enabling the tunnel on iOS 17.0+.
+#if os(iOS)
```

Comment:
This probably shouldn't be in this PR but it's such a tiny change that I included it here. This is to disable the iOS connect on demand workaround on macOS entirely - right now it technically executes.